repositories/src/main/scala/cromwell/pipeline/datastorage/utils/auth/SecurityDirective.scala

```diff
@@ -25,9 +25,8 @@ class SecurityDirective(authConfig: AuthConfig) {
           .map(provide)
           .getOrElse(complete(StatusCodes.Unauthorized -> UnauthorizedMessages.AnotherTypeOfToken))
       case None =>
-        complete(StatusCodes.Unauthorized -> UnauthorizedMessages.MissedToken)
+        reject()
```

Comment:
We must provide custom rejection here, don't we?

Reply:
Done